src/App.js

```diff
@@ -3,7 +3,7 @@ import Mockman from "mockman-js";
 import "react-toastify/dist/ReactToastify.min.css";
 import { Routes, Route } from "react-router-dom";
 import { Header, Sidebar, Search } from "./components/index";
-import { Home, Login, SignUp, Trash } from "./pages/index";
+import { Home, Login, SignUp, Trash, Archives } from "./pages/index";
```

Comment:
Here, if you write path name up to pages then also it will work. No need to mention index explicitly.